src/routes/movement-detection/Sensors.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useCallback } from 'react';
+import withStyles from 'isomorphic-style-loader/withStyles';
+import map from 'lodash/map';
+import Sensor from './Sensor/Sensor';
+import Aside from './Aside/Aside';
+import s from './Sensors.css';
+
+function Sensors() {
+  const [file, setFile] = useState('');
+  const [sensors, setSensors] = useState([]);
```

Comment:
There is `react-perf` issue here: `sensors` are occasionally unnecessarily replaced

Reply:
Consider using `useList` from `react-use` package

Reply:
or, well, rewrite it to use Redux